security/tor-devel/Portfile

```diff
@@ -24,8 +24,8 @@ homepage            https://www.torproject.org/
 master_sites        https://dist.torproject.org/
 distname            tor-${version}
 
-checksums           rmd160  da1d60e1e6a4cfe7f63b266e6cf24f6e1f4ca294 \
-                    sha256  81259e71359b5add9959c0d64430147a939fb12757dc118653bd9c4ed93434e5 \
+checksums           rmd160  04b2406c7c34b9787239f5b76f5ffc775f673a5d \
```

Comment:
CI fails because the checksums don't match. Can you check them again? Is this a stable URL?

Reply:
It was actually the size :)